File: Jazz.web.frame/net/Jazz.Common.Enitiy/IAttribute/DBItemAttribute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Jazz.Common.Enitiy.IAttribute
{
    public class DBItemAttribute:Attribute
    {

        public System.Data.SqlDbType T { get; set; }

        public int Size { get; set; }

        public bool Null { get; set; }

        public bool Check { get; set; }

        public bool Show { get; set; }

        public string Display { get; set; }

        public string ColName { get; set; }
    }
}

File: Jazz.web.frame/java/jazz.common.enitiy/src/org/jazz/common/enitiy/attribute/DBItemAttribute.java
package org.jazz.common.enitiy.attribute;

import java.lang.annotation.*;

/**
 * 数据库实体字段注解
 * @author wjc
 *
 */
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.RUNTIME)
public @interface DBItemAttribute {
	
	   public int Size() default 0;

       public boolean Null() default false;

       public boolean Check() default false;
       
       public boolean Show() default true;

       public String Display();

       public String ColName();
}
